src/generic/listctrl.cpp

```diff
@@ -151,6 +151,34 @@ wxListItemData::wxListItemData(wxListMainWindow *owner)
         m_rect = new wxRect;
 }
 
+// Check if the item is visible
+bool wxGenericListCtrl::IsVisible(long item) const
+{
+    wxRect itemRect, rect;
+    GetItemRect( item, itemRect );
+    rect = GetClientRect();
```

Comment:
```suggestion
    const wxRect rect = GetClientRect();
```
(and remove its declaration above, of course).

It'd be more clear to call it `clientRect`.